src/UbpCrsAdapter.js

```diff
@@ -1,3 +1,4 @@
+import 'babel-polyfill';
```

Comment:
is this import still needed when we have the "babel-polyfill" in the webpack-"entry"-option?
i guess, it should be bundled automatically with the adapter then.